packages/bpk-component-content-cards/src/BpkContentCard.module.scss

```diff
@@ -67,7 +67,7 @@ $bpk-spacing-v2: true;
   }
 
   &--headline {
-    margin-bottom: $bpk-heading-margin-bottom;
+    margin-bottom: bpk-spacing-sm();
```

Comment:
This is actually a different token now, but it was previously using the wrong value so I have fixed that to match with the one in Figma